create_tables/eq_session.sh

```diff
@@ -0,0 +1,10 @@
+sleep 5
```

Comment:
I don't think we need these sleeps in the all scripts. it was just added to the 1st to make sure dynamo was running

Reply:
Looking at `entrypoint.sh` I think all the table scripts are started in parallel in the background before dynamo is started, so they should all need a sleep.

Reply:
ah yes, missed those `&` hiding at the beginning of the line